### PR TITLE
[ID] Add "osu! File Formats/{osb,osk,osz}"

### PR DESCRIPTION
**Pull request by mousesalt**
Add Indonesian translation of https://osu.ppy.sh/wiki/en/osu%21_File_Formats 
`osb`, `osk`, `osz` only at the moment

### CODE REVIEW

File: wiki/osu!_File_Formats/Osz_(file_format)/id.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# .osz (format file)
+
+**.osz** adalah sebuah arsip file yang mengandung tingkat kesulitan beatmap dan sumber daya lainnya, seperti skin maupun hitsounds.
```
Comment (wowcake) [inline wiki/osu!_File_Formats/Osz_(file_format)/id.md:3]:
```suggestion
**.osz** adalah sebuah arsip file yang mengandung tingkat kesulitan beatmap dan sumber daya lainnya, seperti skin maupun hitsound.
```

Comment (wowcake) [reply wiki/osu!_File_Formats/Osz_(file_format)/id.md:3]: remove the s

**Review by wowcake — CHANGES_REQUESTED** [review]

File: wiki/osu!_File_Formats/Osb_(file_format)/id.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# .osb (format file)
+
+.osb adalah sebuah file arsip yang mengandung informasi mengenai tentang storyboard osu!.
```
Comment (ZeroPyrozen) [inline wiki/osu!_File_Formats/Osb_(file_format)/id.md:3]:
```suggestion
.osb adalah sebuah format file yang mengandung informasi mengenai tentang [storyboard osu!](/wiki/Storyboards).
```

File: wiki/osu!_File_Formats/Osk_(file_format)/id.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# .osk (format file)
+
+**.osk** adalah file arsip yang berisi folder dari [skin osu!](/wiki/Skinning).
```
Comment (ZeroPyrozen) [inline wiki/osu!_File_Formats/Osk_(file_format)/id.md:3]:
```suggestion
**.osk** adalah file arsip yang berisi folder [skin osu!](/wiki/Skinning).
```

**Review by ZeroPyrozen — CHANGES_REQUESTED** [review]

**Review by ZeroPyrozen — APPROVED** [review]
All good 👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**